src/python/grpcio/grpc/__init__.py

```diff
@@ -616,7 +646,8 @@ def __call__(self, metadata, error):
 class AuthMetadataPlugin(abc.ABC):
     """A specification for custom authentication."""
 
-    def __call__(self, context, callback):
+    def __call__(self, contex: AuthMetadataContext,
```

Comment:
Actually, now that I think about it, the name of this parameter doesn't matter since it's positional and this is just the interface, not a concrete implementation.